### PR TITLE
Document Guillotine type definitions on NPM

### PR DESCRIPTION
**Pull request by alansemenov**

### CODE REVIEW

File: docs/release.adoc [hunk 1]
```diff
@@ -6,7 +6,35 @@ IMPORTANT: Guillotine 7 requires XP 7.14.0 or higher.
 
 === Release features
 
-SiteKey:: A new `siteKey` argument has been added to the `guillotine` field. This argument is optional and can be used instead of the `X-Guillotine-SiteKey` header.
+Type definitions on NPM:: Guillotine type definitions are now https://www.npmjs.com/package/@enonic-types/guillotine[available on NPM^]. You can install them using the following command:
+
+  npm install @enonic/guillotine-types --save-dev
```
Comment (anatol-sialitski) [inline docs/release.adoc:11]:
```suggestion
  `npm install @enonic/guillotine-types --save-dev`
```

File: docs/release.adoc [hunk 2]
```diff
@@ -6,7 +6,35 @@ IMPORTANT: Guillotine 7 requires XP 7.14.0 or higher.
...
+
+import {ObjectTypeName} from '@enonic-types/guillotine';
+----
+
+siteKey:: A new `siteKey` argument has been added to the `guillotine` field. This argument is optional and can be used instead of the `X-Guillotine-SiteKey` header.
+
+.Retrieve site's displayName and child items using `siteKey` argument.
+[source,graphql]
+----
+{
```
Comment (anatol-sialitski) [inline docs/release.adoc:28]: {
  guillotine(siteKey: "/my-site"){
    get(key: "${site}") {
       	displayName
	    children {
	       displayName
	       type
	    }
    }
  }
}

**Review by anatol-sialitski — CHANGES_REQUESTED** [review]
Please, fix example for siteKey

File: docs/release.adoc (hunk 1 above)
Comment (alansemenov) [reply docs/release.adoc:11]: It's not necessary when there's 2 space indentation. It will render as a code block.

**Pull request merged**